packages/api/src/dwn-api.ts

```diff
@@ -91,6 +106,11 @@ export type RecordsWriteRequest = {
   store?: boolean;
 }
 
+export type PermissionGrantRequest = {
```

Comment:
@diehuxx This type appears to be a duplicate.  It isn't used anywhere as best I can tell.  Should it be removed?